exonum/src/runtime/mod.rs

```diff
@@ -185,18 +205,15 @@ pub trait Runtime: Send + Debug + 'static {
     /// * If panic occurs, the runtime must ensure that it is in a consistent state.
     fn start_service(&mut self, spec: &InstanceSpec) -> Result<(), ExecutionError>;
 
-    /// Configure a service instance with the given parameters.
+    /// Initialize a service instance with the given parameters.
     ///
     /// The configuration parameters passed to the method are discarded immediately.
     /// So the service instance should save them by itself if it is important for
     /// the service business logic.
     ///
-    /// This method is called in two cases:
-    ///
-    /// * After creating a new service instance by the [`start_service`] invocation. In this case,
-    /// if an error during this action occurs, the dispatcher will invoke [`stop_service`].
-    /// Make sure that this invocation will not fail.
-    /// * During the configuration change procedure. [ECR-3306]
+    /// This method is called after creating a new service instance by the [`start_service`].
+    /// In this case, if an error during this action occurs, the dispatcher will invoke
+    /// [`stop_service`]. Make sure that this invocation will not fail.
```

Comment:
What invocation? Of `initialize_service`?

Reply:
Yes.

Reply:
Then it does not make any sense, as in the previous sentence you say that errors are allowed and will be correctly handled by dispatcher.

Reply:
This is just a recomendation.

Reply:
Then the phrase must be `It is recommended to avoid errors returned from the method.`